src/bloqade/ir/location/list.py

```diff
@@ -61,20 +61,24 @@ def __iter__(self):
 
 start = ListOfLocations()
 """
-    - Program starting node
-    - Possible Next <LevelCoupling>
-
-        -> `start.rydberg`
-            :: address rydberg level coupling
-
-        -> `start.hyperfine`
-            :: address hyperfine level coupling
-
-    - Possible Next <AtomArragement>
-
-        -> `start.add_locations(List[Tuple[int]])`
-            :: add multiple atoms to current register
-
-        -> `start.add_location(Tuple[int])`
-            :: add atom to current register
+A Program starting point, alias of empty
+[`ListOfLocations`][bloqade.ir.location.list.ListOfLocations].
+
+- Next possible steps to build your program are:
+- Specify which level coupling to address with:
+    - |_ `start.rydberg`: for [`Rydberg`][bloqade.builder.coupling.Rydberg]
+        Level coupling
+    - |_ `start.hyperfine`: for [`Hyperfine`][bloqade.builder.coupling.Hyperfine]
+        Level coupling
+    - LOCKOUT: You cannot add atoms to your geometry after specifying level coupling.
```

Comment:
markdown has `!!!` blocks for highlighted warning/errors (usually, not sure vscode renders it tho)

Reply:
Ah yes, this should become an admonition inside mkdocs too, right?

Reply:
yes